CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -316,14 +317,15 @@ class Fridge(DB_Item):
         "location/zip": {"required": False, "max_length": 10},
         "location/geoLat": {"required": True, "max_length": 20},
         "location/geoLng": {"required": True, "max_length": 20},
+        "location/country": {"required": False, "max_length": 256},
         "maintainer": {"required": False, "type": "S"},
         "maintainer/name": {"required": False, "max_length": 256},
         "maintainer/organization": {"required": False, "max_length": 256},
         "maintainer/phone": {"required": False, "min_length": 10, "max_length": 20},
         "maintainer/email": {"required": False, "max_length": 320},
         "maintainer/website": {"required": False, "max_length": 2048},
         "maintainer/instagram": {"required": False, "max_length": 64},
-        "notes": {"required": False, "max_length": 280, "type": "S"},
+        "notes": {"required": False, "max_length": 700, "type": "S"},
```

Comment:
increased to 700, it's pretty long but didn't want to cut anyone short. There were only a few fridges that had really lengthy notes section. If we see that this causes the frontend to look too busy we can add a component on the frontend that blocks some of it and the user has to click on "see more" to see the rest of the notes perhaps?